gc/base/HeapRegionManagerTarok.cpp

```diff
@@ -86,6 +86,16 @@ MM_HeapRegionManagerTarok::tearDown(MM_EnvironmentBase *env)
 	MM_HeapRegionManager::tearDown(env);
 }
 
+bool
+MM_HeapRegionManagerTarok::areFreeRegionsForNode(MM_EnvironmentBase *env, uintptr_t numaNode)
+{
+	bool ret = false;
```

Comment:
this could be just one line: return (... != ...);

Reply:
done